webpack/JobWizard/steps/AdvancedFields/AdvancedFields.js

```diff
@@ -42,6 +45,11 @@ export const AdvancedFields = ({ advancedValues, setAdvancedValues }) => {
             }
           />
         )}
+        <DescriptionField
+          inputs={templateInputs}
+          value={advancedValues.description}
+          setValue={newValue => setAdvancedValues({ description: newValue })}
```

Comment:
a cool shortcut I thought about, but just a small nit

```suggestion
          setValue={description => setAdvancedValues({ description })}
```